navigation_onboarding/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.22.0)
+
+set(CMAKE_CXX_COMPILER clang++-12)
```

Comment:
Don't force the compiler. You want your build to be as independent of the compiler used as possible. Case in point: 

```
-- The C compiler identification is GNU 11.2.0
-- The CXX compiler identification is unknown
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
CMake Error at CMakeLists.txt:5 (project):
  The CMAKE_CXX_COMPILER:

    clang++-12

  is not a full path and was not found in the PATH.

  Tell CMake where to find the compiler by setting either the environment
  variable "CXX" or the CMake cache entry CMAKE_CXX_COMPILER to the full path
  to the compiler, or to the compiler name if it is in the PATH.


-- Configuring incomplete, errors occurred!
See also "/home/miltfra/Desktop/workshops-2023/navigation_onboarding/build/CMakeFiles/CMakeOutput.log".
See also "/home/miltfra/Desktop/workshops-2023/navigation_onboarding/build/CMakeFiles/CMakeError.log".
```

Reply:
Yeah I did that to be lazy and forgot about it lol.